pkg/constants/constants.go

```diff
@@ -51,6 +51,14 @@ const (
 	// It is an opaque string, but might be semver.
 	AnnotationNewVersion = Prefix + "new-version"
 
+	// Key set by the update-operator when pre- and post-reboot actions should run
+	LabelRebootAction = Prefix + "reboot-action"
```

Comment:
Again, I didn't think we were actually exposing "action" in the label names. Why not use something the user would expect like:

```
nodeSelector:
    container-linux-update.v1.coreos.com/before-reboot: "true"
```

or

```
nodeSelector:
    container-linux-update.v1.coreos.com/after-reboot: "true"
```